github-stats/fetcher/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: fetcher
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: fetcher
+    spec:
+      containers:
+      - name: fetcher
+        args:
+        - --token-file=/etc/secret-volume/token
+        - --stderrthreshold=0
+        - --host=sqlproxy-service
+        image: gcr.io/google-containers/fetcher
```

Comment:
Please use a specific version. Alternatively name this deployment-dev.yaml to make it clear this is not the production version (and if so please create a production yaml)